cmake/bink.cmake

```diff
@@ -0,0 +1,9 @@
+set(BINK_DIR ${CMAKE_SOURCE_DIR}/Dependencies/.bink)
+
+FetchContent_Declare(
+    bink
+    GIT_REPOSITORY https://github.com/TheSuperHackers/bink-sdk-stub.git
+    GIT_TAG        master
```

Comment:
You must define a specific tag or commit hash or it will be a mess to reproduce builds

Reply:
Fixed.